src/libcollections/macros.rs

```diff
@@ -9,12 +9,30 @@
 // except according to those terms.
 
 /// Creates a `Vec` containing the arguments.
+///
+/// `vec!` allows `Vec`s to be defined analogous to array expressions. There
```

Comment:
Sentence is a bit awkard. Maybe instead of "analogous to" use "with the same syntax as".